src/restLevelClient/generateClientDefinition.ts

```diff
@@ -116,14 +116,6 @@ export function generatePathFirstClient(model: CodeModel, project: Project) {
   });
 
   const clientName = getLanguageMetadata(model.language).name;
-  const uriParameter = getClientUriParameter();
-
-  const { addCredentials, credentialKeyHeaderName } = getAutorestOptions();
-  const credentialTypes = addCredentials ? ["TokenCredential"] : [];
-
-  if (credentialKeyHeaderName) {
-    credentialTypes.push("KeyCredential");
-  }
```

Comment:
Are we no longer supporting KeyCredential?

Reply:
No. I delete it in generating `clientDefinitions`, which doesn't need to import credential related packages.